.github/workflows/test-unreleased.yml

```diff
@@ -53,14 +87,32 @@ jobs:
           ruby -v
           rake bundle:clean
           rake bundle:install
+        working-directory: ruby-client
 
       - name: opensearch-ruby
```

Comment:
For another time, make we can have a top level Rakefile to build and test, since I don't think we want gemspecs that are wildly incompatible across these gems anyway?

Reply:
We have to `working-directory: ruby-client` because the client was cloned into `ruby-client` folder to avoid collision with the `opensearch` repo that is also being pulled into the same machine. We do have a top level Rakefile and Gemfile.